passes/cmds/logger.cc

```diff
@@ -58,7 +58,8 @@ struct LoggerPass : public Pass {
 		log("        do not print warnings for the specified experimental feature\n");
 		log("\n");
 		log("    -expect <type> <regex> <expected_count>\n");
-		log("        expect log,warning or error to appear. In case of error return code is 0.\n");
+		log("        expect log, warning or error to appear. matched errors will terminate\n");
```

Comment:
```suggestion
		log("        expect log, warning or error to appear. Matched errors will terminate\n");
```

Actually, the general lack of capitalization and the strange grammar in the help message bother me.  I will submit a separate PR...

Reply:
I think if you look across most of Yosys' docs it does not capitalise the beginning of a paragraph, nor after full stops. I tried to be rebellious and fix that in the past, but now I don't bother and just comply (and even revert) to the majority.

Reply:
Maybe we can start a revolution :).